src/utils/utils.js

```diff
@@ -2,3 +2,12 @@ export const formatDate = dateString => {
   const date = new Date(dateString)
   return `${date.getDate()}.${date.getMonth() + 1}.${date.getFullYear()}`
 }
+
+export const getLangCode = slug => {
+  const regex = /(\bfi)|(\ben)|(\bsv)/g
```

Comment:
Will `en` ever be in the slug, or will the lang code always be omitted? If it is omitted in case the lang is en, I think the comment below should be checked so we don't end up for example in `test.io/my-sv-event` going to `sv` although the lang is en?

Reply:
it is - when generating the events, they all have the formula of `/langCode/name-of-the-event`, so english ones have the lang code in their name (this is due to the routing thing, and that these events are under the `en`-folder)